### PR TITLE
noisy report alert

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?243866#1261207

@czue seemed like having an alert for reports getting dropped is useful, but this should give us 10 emails instead of 1000

@proteusvacuum

### CODE REVIEW

File: corehq/apps/reports/scheduled.py [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+from corehq.util.soft_assert import soft_assert
+
+
+_soft_assert = soft_assert(exponential_backoff=True)
 
 
 def get_scheduled_reports(period, as_of=None):
     as_of = as_of or datetime.utcnow()
-    assert period in ('daily', 'weekly', 'monthly')
+    assert period in ('daily', 'weekly', 'monthly'), period
```
Comment (emord) [inline corehq/apps/reports/scheduled.py]: @millerdev pr followup 2 years after the fact :) https://github.com/dimagi/commcare-hq/pull/4285/files#r17305879

Comment (czue) [inline corehq/apps/reports/scheduled.py]: hahaha my bad. thanks for the :mag_right:

**Review by czue — APPROVED** [review]
thanks

File: corehq/apps/reports/scheduled.py (hunk 1 above)
Comment (emord) [inline corehq/apps/reports/scheduled.py]: Ha. cory linked the commit for some context in the interrupt ticket. figured I'd just follow up. Don't think it's ever been triggered

**Pull request merged**